sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/TestEnvironment.cs

```diff
@@ -21,35 +23,58 @@ public static class TestEnvironment
         private const string AssetsFolderName = "Assets";
 
         /// <summary>The name of the JPG file which contains the receipt to be used for tests.</summary>
-        private const string ReceiptFilename = "contoso-receipt.jpg";
+        private const string JpgReceiptFilename = "contoso-receipt.jpg";
+
+        /// <summary>The name of the PNG file which contains the receipt to be used for tests.</summary>
+        private const string PngReceiptFilename = "contoso-allinone.png";
 
         /// <summary>The format to generate the filenames of the PDF forms to be used for tests.</summary>
```

Comment:
Comment bug calling this PDF?

Will we have invoice types for every file format?

Reply:
Yep, my mistake. I'll get it fixed. Regarding the invoice types, I don't have any specific plan in mind, so it will depend on what tests we decide to add. I'm using a TIFF invoice file because that's what was available in our data samples (and it was present in python test resources as well, if I'm not mistaken). If we decide to keep only PDF and TIFF in the long term, we can simplify the logic here a bit.

Reply:
Feel free to take what I have: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer/tests/sample_forms